composer.json

```diff
@@ -6,22 +6,22 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "drupal/core": "^8.8",
-        "php": ">=7.2"
+        "drupal/core": "^8.9 || ^9",
+        "php": ">=7.3"
     },
     "require-dev": {
         "composer/installers": "~1.5",
-        "drupal/core-composer-scaffold": "^8.8",
+        "drupal/core-composer-scaffold": "^8.9 || ^9",
         "drupal/config_devel": "~1.2",
         "drupal/drupal-extension": "~4.0",
         "drupal/tmgmt": "1.11",
-        "drush/drush": "~9.0@stable",
+        "drush/drush": "~10.3",
         "guzzlehttp/guzzle": "~6.3",
         "nikic/php-parser": "~4.10.4",
-        "openeuropa/code-review": "~1.5",
-        "openeuropa/drupal-core-require-dev": "^8.8",
-        "openeuropa/task-runner": "~1.0-beta5",
-        "phpunit/phpunit": "~6.0"
+        "openeuropa/code-review": "~1.6",
+        "openeuropa/drupal-core-require-dev": "^8.9 || ^9",
+        "openeuropa/task-runner-drupal-project-symlink": "^1.0",
+        "phpunit/phpunit": "^7 || ^8"
```

Comment:
Why not allowing also PHPUnit ^9?

Reply:
I followed the drupal 9 compatibility fixes on previous modules, and since we are supporting only php 7.3, drupal 9 installations running on php 7.3 only uses PHPUnit 8 (see https://www.drupal.org/node/3176567)

Reply:
We're not supporting ONLY PHP 7.3, we're supporting >=7.3, that meaning also PHP 7.4 and 8.0, see https://jubianchi.github.io/semver-check/#/%3E%3D7.3/8.0.0.

If one wants to contribute to this module and he/she is on PHP 8, we shouldn't limit their choice. The same goes down, Drupal 8.9 still accepts PHPUnit ^6.5. So we need `^6.5 || ^7 || ^8 || ^9`